From: Eric Dumazet <eric.dumazet@gmail.com>
Date: Sat, 10 Apr 2010 22:48:14 +0000
Subject: [PATCH 7/22] drivers: net: last_rx elimination
References: bnc#674189, fate#311981
Patch-mainline: v2.6.35-rc1
Git-commit: 127aa0e14b3e1f8476c4781facdfac11d5546872

Network drivers do not have to update last_rx, unless they need it for
their private use.

Signed-off-by: Eric Dumazet <eric.dumazet@gmail.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Chen, Chien-Chia <machen@suse.com>
---
 drivers/net/vmxnet3/vmxnet3_drv.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/net/vmxnet3/vmxnet3_drv.c
+++ b/drivers/net/vmxnet3/vmxnet3_drv.c
@@ -1183,7 +1183,6 @@ vmxnet3_rq_rx_complete(struct vmxnet3_rx
 				netif_receive_skb(skb);
 			}
 
-			adapter->netdev->last_rx = jiffies;
 			ctx->skb = NULL;
 		}
 
